app/src/routes/+layout.svelte

```diff
@@ -57,11 +60,26 @@
 	setContext(LineManagerFactory, data.lineManagerFactory);
 	setContext(ProjectMetrics, data.projectMetrics);
 
+	const user = data.userService.user;
+	const accessToken = $derived($user?.github_access_token);
+	const octokit = $derived(octokitFromAccessToken(accessToken));
+
+	// This store is literally only used once, on GitHub oauth, to set the
+	// gh username on the user object. Furthermore, it isn't used anywhere.
+	// TODO: Remove the gh username completely?
+	const githubUserService = $derived(octokit ? new GitHubUserService(octokit) : undefined);
```

Comment:
Because we're always creating a octokit even when the accessToken is undefined, this will always create teh GitHubUserService